src/app/credential-issuer/router.js

```diff
@@ -1,12 +1,10 @@
 const express = require("express");
-const csrf = require("csurf");
 
 const router = express.Router();
 
 const { sendParamsToAPI, sendParamsToAPIV2 } = require("./middleware");
-const csrfProtection = csrf({});
 
-router.get("/callback", csrfProtection, sendParamsToAPI);
-router.get("/callback/:criId", csrfProtection, sendParamsToAPIV2);
```

Comment:
Adding the csrf function to the app means that we don't need to add it to individual routes